tensorflow/compiler/xla/stream_executor/rocm/roctracer_wrapper.h

```diff
@@ -23,11 +23,7 @@ limitations under the License.
 #include "rocm/include/roctracer/roctracer.h"
 #include "rocm/include/roctracer/roctracer_hcc.h"
 #include "rocm/include/roctracer/roctracer_hip.h"
-<<<<<<< HEAD
-#include "rocm/include/roctracer/roctracer_roctx.h"
```

Comment:
i would double check if removing `#include "rocm/include/roctracer/roctracer_roctx.h"` is correct.

Reply:
Yes, I added it. You're reviewing my commit that resolve conflict. Please check my commit to fix build error.